maxun-core/src/utils/worker.ts

```
@@ -0,0 +1,157 @@
+                await page.waitForTimeout(1000);
+
+            } catch (error) {
+                parentPort?.postMessage({
+                    type: 'error',
+                    data: {
+                        workerId: config.workerIndex,
+                        url: pageUrl,
+                        error: error.message
+                    }
+                });
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)